packages/elements/src/Text/utils/styles/addMarginsToLists.ts

```diff
@@ -0,0 +1,34 @@
+const listMargins = (node: Element): void => {
+  const allowedTags = ["UL", "OL"];
+  if (allowedTags.includes(node.nodeName)) {
+    const { marginTop, marginBottom } = window.getComputedStyle(node);
+
+    if (marginTop) {
+      node.firstElementChild?.classList.add(
+        `brz-mt-lg-${marginTop.replace(/\D/g, "")}`
```

Comment:
marginTop can be `auto`
![image](https://github.com/bagrinsergiu/MB-migration/assets/18303258/a4967423-1153-474c-8506-30cfe8f98f06)

> maybe use `parseInt()` them check to `NaN`